mod/wikirate/set/right/citation_count.rb

```diff
@@ -32,6 +39,6 @@ def no_citations? args
   end
 
   def related_overview_card
-    Card.fetch "#{card.cardname.left}+#{Card[:related_articles]}"
+    Card.fetch "#{card.cardname.left}+#{Card[:related_articles].name}"
```

Comment:
my favorite syntax for this (and actually slightly faster):
`Card.fetch card.cardname.left_name.trait(:related_articles)`